docs/usage-exapmle.md

```diff
@@ -36,7 +37,13 @@ fun main(args: Array<String>) {
 ```ts
   const greeting = () => console.log("Hello, world!");
 ```
-<button>Try demo</button>
-<button className="button-secondary">Try demo</button>
-<button className="button-ghost">Try demo</button>
+<button className="button-primary">
+    <img src="../static/img/download-icon.svg" alt=""/>
```

Comment:
Please, do not use `<img>` tag for icons
I suggest implementing `<Icon src="../static/img/download-icon.svg">` component